lastuser_core/models/user.py

```diff
@@ -956,7 +959,7 @@ def __repr__(self):
         )
 
     def __unicode__(self):
```

Comment:
The `__unicode__` methods can be removed as these are just `__str__` now. Multiple instances below.